cp/richTextFieldWithTables/v1/index.js

```diff
@@ -128,6 +128,11 @@ Appian.Component.onNewValue(function (allParameters) {
 
   // Always set the Appian value after setting the editor content to pass back out the formatted html (this will only run if the value actually changed)
   setAppianValue();
+
+  // After initialization, run function to update styles for accessibility compliance
+  setA11yCss();
```

Comment:
@annikaBasch this line should be right below `setDynamicCss()` (line 112) since that stuff only needs to run when the editor is created, not on every interaction.